client/src/App.js

```diff
@@ -290,82 +313,96 @@ function App () {
-                    disabled={disableGuessBox || isOnscreenKeyboardVisible}
-                    label="Guess!"
-                    value={guess}
-                    variant="outlined"
-                    onKeyDown={(e) => guessBoxPressed(e)}
-                    onChange={(e) => {
-                      setGuess(e.target.value);
-                      if (keyboardRef.current) {
-                        keyboardRef.current.setInput(e.target.value);
-                      }
-                    }}
-                  />
-                ) : (
-                  <Typography variant="h3">{drawWord}</Typography>
-                )}
-              </div>
+            </Typography>
+          </Toolbar>
+        </AppBar>
+        <Grid container className="layoutContainer">
+          <Hidden mdDown>
+            <Grid item md={3} lg={3}>
+              <UserScoreList userScores={userScores} />
+              <LogWindow className="logWindow" messages={messageLog}></LogWindow>
             </Grid>
-            {renderKeyboard()}
-            {previousWord ? (
+          </Hidden>
+          <Grid item md={12} lg={6}>
+            <Grid container>
               <Grid item xs={12}>
-                <Typography variant="body1">
-                  Previous word was {previousWord}
-                </Typography>
+                <Paper elevation={3} className="canvasContainer">
+                  {renderPlayersIcon()}
+                  {renderGameState()}
+                </Paper>
               </Grid>
-            ) : null}
+              <Grid item xs={12}>
+                <div className="inputContainer">
+                  {showGuessBox ? (
+                    <TextField
+                      className="guessBox"
+                      id="txt-guess"
+                      size="small"
+                      ref={guessBoxRef}
+                      disabled={disableGuessBox || isOnscreenKeyboardVisible}
+                      label="Guess!"
+                      value={guess}
+                      variant="outlined"
+                      onKeyDown={(e) => guessBoxPressed(e)}
+                      onChange={(e) => {
+                        setGuess(e.target.value);
+                        if (keyboardRef.current) {
+                          keyboardRef.current.setInput(e.target.value);
+                        }
+                      }}
+                    />
+                  ) : (
+                    <Typography variant="h5">{drawWord}</Typography>
+                  )}
+                </div>
+              </Grid>
+              {renderKeyboard()}
+              {previousWord ? (
+                <Grid item xs={12}>
+                  <Typography variant="body1">
+                  Previous word was {previousWord}
+                  </Typography>
+                </Grid>
+              ) : null}
+            </Grid>
+          </Grid>
+          <Grid item xs={12} sm={12} md={12} lg={3}>
+            {renderCanvasToolbox()}
```

Comment:
We may have to eventually move the toolbox inside game area to free up App.js. But for now I can't think of how.